capstone-project

Tadessejemal1 opened the pull request:
**create a capstone project**
**Project requirements**

1. I personalize the content of my page and I Choose a Robotics topic 
2. I followed these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),
3. I build 2 pages: Home and About
4. create a mobile menu interaction and Dynamic page

Tadessejemal1 committed:
capstone-project

Tadessejemal1 committed:
nav bar header

Tadessejemal1 committed:
_config.yml

Tadessejemal1 committed:
progress commit

Tadessejemal1 committed:
Add styles about home page

Tadessejemal1 committed:
fix linters

Tadessejemal1 committed:
fix linters

Tadessejemal1 committed:
fix linters

Tadessejemal1 committed:
fix stylintrc and eslint errors

Tadessejemal1 committed:
Add html,css and javascript on project

tresorsawasawa commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
- [ **OPTIONAL** ] I think it would be better if you add the `.vscode/` directory in the `.gitignore` file as it has content non-related to the project, once you did it you can now remove it from the Github repo content.

### In the [README.md](https://github.com/Tadessejemal1/FIRST-CAPSTONE-PROJECT/blob/first-capstone-project/README.md) file :point_down: 
- [ ] Kindly make sure you complete your README.md file professionally as required for the project (i.e. each section should have the expected information) e.g.: meaningful project's title, meaningful description, a link to the online page(Live Demo), information about cloning the project, set up the project, information about the author,... **To help you complete your README, I suggest you use this [TEMPLATE](https://github.com/tresorsawasawa/Solidarity-Up/blob/main/README.md)**

- [ ] Kindly make sure your record a **[Loom](https://www.loom.com/)** video with the following feature and once you did it, make sure you add it to the README file(in its specific section) 
The **[Loom](https://www.loom.com/)** should contain the below features:
  - A video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, showing the pages in different screen sizes.
  - The video contains a highlight about a piece of the code you find interesting or is proud of making.

- [ ] Once you update your **README**, make sure it contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) and required for this project.

- [ ] As required for the project, kindly make sure your project has its Live version and you use GitHub Pages to deploy the web pages. i.e. the project is deployed properly, all pages, links, and images work as expected.

- [ ] As I mentioned in one of the previous comments a link to the deployed project should be present in the README(Live Demo section).

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+        <div></div>
+      </div>
+    </div>
+      <div id="overlay">
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="#programs">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsers">Sponsors</a>
+          <a href="./about.html">About</a>
+          <a href="#">
+            <span class="cc-campaign">Robo-Campaign</span>
+          </a>
+      </div>
```
- [ ] Kindly make sure you fix the issue of overlapping elements on your mobile menu content, on the hamburger button], and close the menu button when scrolling as shown in the images below, both on the about and home pages to make your project looks professional.

| Home page | About page |
| ------------------------------ | -------------------------------------- |
| ![Screenshot from 2022-07-28 02-28-04](https://user-images.githubusercontent.com/83042742/181394995-51b89125-fb6d-4386-ab07-b098e2d5ca19.png) | ![Screenshot from 2022-07-28 02-23-59](https://user-images.githubusercontent.com/83042742/181395141-73fe2842-e5ff-4497-b45f-1e46efc85702.png) |
| ![Screenshot from 2022-07-28 02-27-29](https://user-images.githubusercontent.com/83042742/181395035-d9fba7f3-9468-48fb-9080-e80654c30fdc.png) |
 
| ![Screenshot from 2022-07-28 02-27-29](https://user-images.githubusercontent.com/83042742/181395035-d9fba7f3-9468-48fb-9080-e80654c30fdc.png) |

tresorsawasawa replied:
- [ ] For good semantics tag and as mentioned in the previous comment, I suggest you wrap all this block in a `<nav id="overlay">` tag as the container instead of a `div`,  all the navigation options in a `ul` tag and each option link wrapped in a `li` tag, i.e.:
```
   <nav id="overlay">
      <ul class="...">
        <li>
            <a href="./index.html" class="home-link">Home</a>
         </li>
         <li>
            <a href="#programs">Program</a>
         </li>
         <li>
             <a href="#speakers">Speakers</a>
          </li>
         ...

       </ul>
    </nav>
```

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+      <div id="hamburger">
+        <div></div>
+      </div>
+    </div>
```
- [ ] Kindly make sure your hamburger button follows the same styling as on the one from the**[template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** especially with the color and the background

| Template design | Your Design |
| --------------------- | ----------------- |
| ![Screenshot from 2022-07-28 02-41-51](https://user-images.githubusercontent.com/83042742/181396309-39c8be24-8eb7-4d83-941b-355f1cf81420.png) | ![Screenshot from 2022-07-28 02-41-12](https://user-images.githubusercontent.com/83042742/181396328-7dceff40-da7a-4ab2-a711-67f302e75801.png) |

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+        </p>
+        <h3>June 27-28, 2022 Addis Ababa, Ethiopia</h3>
+        <p class="location">
+          &#64; 4-kilo main hall, Addis Ababa University.
```
- [ ] Kindly make sure the correct color text like on the **Behance** template in order to stick to the expected design.

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+    <!-- Footer -->
+    <section class="footer">
+      <div class="footer-main">
+        <div class="footer-image">
+          <img src="./images/logo 2.jpg" alt="footer-image" />
+        </div>
```
- [ **OPTIONAL** ] What do you think if you use your logo as a link so it can be used to be redirected to the home section to make your project more personal.

tresorsawasawa commented on about.html:
```diff
@@ -0,0 +1,191 @@
+      <!-- Footer -->
+      <section class="footer">
+        <div class="foot"></div>
+        <div class="footer-main">
+          <div class="footer-image">
+            <img src="./images/logo 2.jpg" alt="footer-image" />
+          </div>
```
- [ **OPTIONAL** ] What do you think if you use your logo as a link so it can be used to be redirected to the home section to make your project more ptresional.

tresorsawasawa commented on about.html:
```diff
@@ -0,0 +1,191 @@
+    <title>Capstone Project About Pages</title>
+</head>
+<body>
+  <header>
+    <!-- Nav-bar -->
+    <div class="container">
+      <div class="header_nav">
+        <a href="https://www.facebook.com/"
+          ><i class="fa-brands fa-facebook-f"></i
+        ></a>
+        <a href="https://twitter.com/"
+          ><i class="fa-brands fa-twitter"></i
+        ></a>
+        <a href="#">English</a>
+        <a href="#">My Pages</a>
+        <a href="#">Logout</a>
+      </div>
+    </div>
+    <div class="nav">
+      <div id="hamburger">
+        <div></div>
+      </div>
+    </div>
+    <div id="overlay">
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="#third-section">Program</a>
+          <a href="#news">News</a>
+          <a href="#container">Join</a>
+          <a href="./index.html">Home</a>
+          <a href="#">
+            <span class="cc-campaign">Robo-Campaign</span>
+          </a>
+    </div>
+    <a href="#" class="logo"
+      ><img src="./images/Capture-logo.JPG" alt="nav-logo"
+       /></a>
```
- [ ] For good tags semantics I suggest you delimitate your header section correctly and remove the **closed header tag** (`</header>`) at the bottom of the page, Line 188,(as all the content of the page is not part of the header section), then put it at it expected place.

tresorsawasawa commented on about.html:
```diff
@@ -0,0 +1,191 @@
+            </p>
+          </div>
+        </div>
+      </section>
```
- [ **OPTIONAL** ] **RECOMENDED:** For good tags semantics, I suggest you wrap this block in a `footer` tag instead of a `section` tag to make your code base looks more professional.

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+        <a href="#">My Pages</a>
+        <a href="#">Logout</a>
+      </div>
+    </div>
```
- [ ] For good semantics tag, I suggest you wrap all this block in a `<nav class="container">` tag as the container instead of a `div`, then then, `header_nav` in a `ul` tag and each link wrapped in a `li` tag i.e.:
```
   <nav class="container">
      <ul class="header_nav">
        <li>
             <a href="https://www.facebook.com/"><i class="fa-brands fa-facebook-f"></i></a>
         </li>
          ...
          <li>
             <a href="#">English</a>
          </li>
         ...
       </ul>
    </nav>
```

tresorsawasawa commented on styles.css:
```diff
@@ -1,3 +1,861 @@
+}
+
+@media (min-width: 768px) {
+  /* .no-scroll {
+    overflow-y: auto;
+    overflow-x: hidden;
+  } */
+
```
- [ ] Kindly make sure you remove all the commented code lines to make your code base cleaner.

tresorsawasawa commented on index.html:
```diff
@@ -5,25 +5,197 @@
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-3.svg"
+            alt="icon-3"
+            class="item-icon"
+          />
+          <h3 class="program-title">Applications</h3>
+          <p class="program-description">
+            RI is now applied in almost all robot tasks, including manufacturing
+            , space, aviation, undersea, surgery, rehabilitation, agriculture, education.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-4.svg"
+            alt="icon-4"
+            class="item-icon"
+          />
+          <h3 class="program-title">Human supervisory control</h3>
+          <p class="program-description">
+            robots in performance of routine tasks. These include 
+            handling of parts on manufacturing assembly lines.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-5.svg"
+            alt="icon-5"
+            class="item-icon"
+          />
+          <h3 class="program-title">Human–robot social interaction</h3>
+          <p class="program-description">
+            robot devices to provide entertainment, teaching, comfort, 
+            and assistance for children and elderly, autistic, and handicapped persons.
+          </p>
+        </div>
+      </div>
+      <a href="#" class="conference-held"> <span>Join BDU International conference 2022</span></a>
+    </section>
```
- [ ] As from the Behance template, kindly make sure when a card has hovered, it should have an interactive styling (e.g.: a border, increased size,...)
### Figma Template image
![Screenshot from 2022-07-28 03-37-41](https://user-images.githubusercontent.com/83042742/181401618-da28b91f-1804-45b6-8d7a-49a2f4f2cc19.png)

tresorsawasawa reviewed (commented):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Tadessejemal1 :wave: 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tresorsawasawa reviewed (changes requested):
**Hello**, 
This is here by error.
I am sorry

Tadessejemal1 committed:
Updated README based on the feedback by PR

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
Add styles on nav

Tadessejemal1 committed:
fix navbar styles

Tadessejemal1 committed:
Merge branch 'first-capstone-project' of https://github.com/Tadessejemal1/FIRST-CAPSTONE-PROJECT into first-capstone-project

Tadessejemal1 committed:
fix the project based on PR comments

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
Updated README

Tadessejemal1 committed:
Updated README

Tadessejemal1 commented:
based on the comments of the PR I have Corrected all things this is:

1. Updated README file
2. Create Deployment pages
3. as he mentioned  I fix the issue of overlapping elements on your mobile menu content, on the hamburger button]
this is a screenshot image

 ![image](https://user-images.githubusercontent.com/85793292/181610507-b5d401df-a70f-447c-bdbe-ba48395f9d5c.png)
4. I fix the hamburger button following the same styling as on the one from the**[template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** especially with the color and the background.
There are the images

## my Work 
![image](https://user-images.githubusercontent.com/85793292/181613806-ee2f33b2-74c9-4c08-832d-5bbcf526e4b0.png)

## design page
![image](https://user-images.githubusercontent.com/85793292/181614006-9174af0a-23c3-4b5a-99ef-345be7d80a57.png)

5. he mentioned before wrap all these blocks in a <nav class="container"> tag as the container instead of a div, then then, header_nav in a ul tag and each link wrapped in a li tag i.e.: I fix based on the comments the codes are:

```
<header>
    <!-- Nav-bar-upper -->
        <nav class="container">
          <ul class="header_nav">
            <li>
              <a href="https://www.facebook.com/"
                ><i class="fa-brands fa-facebook-f"></i
              ></a>
            </li>
            <li>
              <a href="https://twitter.com/"
                ><i class="fa-brands fa-twitter"></i
              ></a>
            </li>
            <li><a href="#">English</a></li>
            <li><a href="#">My Pages</a></li>
            <li><a href="#">Logout</a></li>
          </ul>
        </nav>
    <!-- Nav-bar-lower -->   
        <nav class="navbar">
          <a href="#"><img src="./images/Capture-logo.JPG" class="nav-logo" /></a>
          <div class="hamburger">
            <span class="bar"></span>
            <span class="bar"></span>
            <span class="bar"></span>
          </div>
          <ul class="nav-menu">
            <li class="nav-item"><a href="./index.html" class="home-link">Home</a><hr class="text-divide"></li>
            <li class="nav-item"><a href="#programs" class="nav-link">Program</a><hr class="text-divide"></li>
            <li class="nav-item"><a href="#speakers" class="nav-link">Speakers</a><hr class="text-divide"></li>
            <li class="nav-item"><a href="#sponsers" class="nav-link">Sponsors</a><hr class="text-divide"></li>
            <li class="nav-item"><a href="./about.html" class="nav-link">About</a><hr class="text-divide"></li>
            <li class="nav-item" >
              <a href="#">
                <span class="cc-campaign">Robo-Campaign</span>
              </a>
            </li>
          </ul>
        </nav>
    </header>
```

6. as he mentioned when a card has hovered, it should have an interactive styling (e.g.: a border, increased size,...) so I fix based on the comment. this is a screenshot image 
![Screenshot (165)](https://user-images.githubusercontent.com/85793292/181612587-44b58b09-8d5c-4208-946d-af1b35f651f7.png).
Finally, I record a loom video this is the link https://loom.com/share/12babf0838fa44b28213439ed0599276
for further information, I made a video that explains where I fix everything based on the PR comments. this is the link https://loom.com/share/f847c4b8296c45539eee33e3478d5a7a 
Thank You.

Tadessejemal1 committed:
fix everything based on PR comments

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
Update README.md

Tadessejemal1 committed:
fix everything based on PR comments

elyor-doniyorov reviewed (approved):
Hi @Tadessejemal1, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
